perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -422,7 +383,7 @@ def _GenerateCreateCommand(self, ssh_keys_path):
     args = []
     if self.host or self.min_cpu_platform:
       args = ['alpha']
-    elif self.gpus:
+    elif self.gpu_count:
```

Comment:
Thanks, good catch. GPU attachment is supported in alpha, so I will just use alpha for that as well so that both options can be provided.